spec/controllers/catalog_controller_spec.rb

```diff
@@ -221,5 +236,23 @@
       post :update, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
       expect(response.body).to include(druid_no_pre)
     end
+    context 'can take two forms of druid input' do
+      before do
+        post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
+      end
+      let(:druid) { 'druid:jj925bx9565' }
+      let(:druid_no_pre) { 'jj925bx9565' }
```

Comment:
just declare these once and use them for both Patch and Post?  with better names?

Reply:
i could declare it at the top but we already have a bare_druid and prefixed_druid (bj102hs9687) being declared. 
i'm trying to think of a name that I could use that is not `bare_druid` or `prefixed_druid` at the top...can't seem to think of a good one